UPGRADE-6.3.md

```diff
@@ -220,6 +220,103 @@ German date format. For backwards compatibility reasons they work with German da
 - Changed method `WriteLogEntry()`: parameter `$sLogFileName` is now ignored.
 - Changed method `WriteLogEntrySimple()`: parameter `$sLogFileName` is now ignored.
 
+## Backend Theme Library
+
+The Backend was upgraded to Bootstrap 4.1.3.
+
+See the [Bootstrap Migration Guide](https://getbootstrap.com/docs/4.1/migration/) for needed changes to your backend modules.
+
+During the upgrade to Bootstrap 4 the following styles where checked and these are our findings:
```

Comment:
What kind of findings?

Something exists in these locations? Or was changed? To where?

findings -> changes?

Reply:
Changed wording.

Reply:
There is currently only one commit.